cloudformation/antmedia-aws-autoscale-remote_db-template.yaml

```diff
@@ -498,13 +498,12 @@ Resources:
             #!/bin/bash
             touch /usr/local/antmedia/conf/initialized
             bash /usr/local/antmedia/change_server_mode.sh cluster ${DBInstance.PrivateIp}
-            apt-get update 
-            apt-get install -y python3-pip
-            apt-get install -y python3-setuptools
+            sudo apt-get update 
+            sudo apt-get -y install python3-pip
             mkdir -p /opt/aws/bin
-            wget https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz
-            python3 -m easy_install --script-dir /opt/aws/bin aws-cfn-bootstrap-py3-latest.tar.gz
-            /opt/aws/bin/cfn-signal -e $? --stack ${AWS::StackName} --resource OriginGroup --region ${AWS::Region}
+            sudo pip3 install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz --break-system-packages
+            sudo ln -s /usr/local/init/ubuntu/cfn-hup /etc/init.d/cfn-hup
+            /usr/local/bin/cfn-signal -e $? --stack ${AWS::StackName} --resource EdgeGroup --region ${AWS::Region}
```

Comment:
"OriginGroup" has changed to "EdgeGroup" and there  is another "EdgeGroup" below. Are you sure to have this change?